packs/equipment/wooden-nickel.json

```diff
@@ -11,7 +11,7 @@
         "containerId": null,
         "damage": null,
         "description": {
-            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">3</span> envision; Prerequisites You are a master in Deception.</p>\n<hr />\n<p>This rough wooden coin hangs from a string drilled through a hole at its center. When you Activate the wooden nickel, for 10 minutes, you can cause any object you touch of 2 Bulk or less to look, feel, and smell like valuables of a similar size crafted from a precious metal of your choice.</p>\n<p>A creature that touches or Interacts with an affected object can attempt a Perception check against your Deception DC to disbelieve the illusion, and a successful check against your Deception DC with Crafting or an appropriate Lore skill will also reveal the item's true nature. After 10 minutes have elapsed since you Activated the wooden nickel, all affected objects revert to their true form.</p>"
+            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">3</span> envision</p>\n<p><strong>Prerequisites</strong> You are a master in Deception</p><hr /><p>This rough wooden coin hangs from a string drilled through a hole at its center. When you Activate the <em>wooden nickel</em>, for 10 minutes, you can cause any object you touch of 2 Bulk or less to look, feel, and smell like valuables of a similar size crafted from a precious metal of your choice.</p>\n<p>A creature that touches or Interacts with an affected object can attempt a @Check[perception] check against your Deception DC to disbelieve the illusion, and a successful check against your Deception DC with @Check[crafting] or an appropriate Lore skill will also reveal the item's true nature. After 10 minutes have elapsed since you Activated the <em>wooden nickel</em>, all affected objects revert to their true form.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">3</span> envision</p>\n<p><strong>Prerequisites</strong> You are a master in Deception</p><hr /><p>This rough wooden coin hangs from a string drilled through a hole at its center. When you Activate the <em>wooden nickel</em>, for 10 minutes, you can cause any object you touch of 2 Bulk or less to look, feel, and smell like valuables of a similar size crafted from a precious metal of your choice.</p>\n<p>A creature that touches or Interacts with an affected object can attempt a @Check[perception|against:deception|rollerRole:target] check against your Deception DC to disbelieve the illusion, and a successful check against your Deception DC with @Check[crafting|against:deception|rollerRole:target] or an appropriate Lore skill will also reveal the item's true nature. After 10 minutes have elapsed since you Activated the <em>wooden nickel</em>, all affected objects revert to their true form.</p>"
```